### PR TITLE
Updated "Health Report" tab

### PR DESCRIPTION
**Pull request by vchu22**
Added survey menu and questionnaire for each category

### CODE REVIEW

File: src/views/pages/questionnaire/questions/city_state_decisions.json [hunk 1]
```diff
@@ -0,0 +1,613 @@
+{
+  "name": "Sentiment on Your City/State Decisions",
+  "clusters": [
+    {
+      "id": "536bec4fec74e2d688bac731755e50e5fb2cb191832f82269dce4d02940ae607",
```
Comment (Murkantilism) [inline src/views/pages/questionnaire/questions/city_state_decisions.json:5]: What do these id's represent? Are they the ID's of tripetto elements?

File: src/views/pages/questionnaire/questions/utils.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  },
+};
+
+const extractAnswers = (
+  fields,
+  multiple_choice_fields = [],
+  multi_items_string_fields = [],
+  date_fields = [],
+  numeric_fields = []
+) => {
```
Comment (Murkantilism) [inline src/views/pages/questionnaire/questions/utils.js:120]: Is this the function that the above TODO ([here](https://github.com/CitizenHealth/COVIDx/pull/43/files?diff=unified&w=1#diff-de652b6119a63882adf4e07cbf2af114R27-R29)) is referencing?

File: src/views/pages/questionnaire/questions/utils.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  });
+
+  // process numeric fields
+  numeric_fields.forEach((field) => {
+    if (finalResult[field] !== undefined) {
+      if (finalResult[field].includes("."))
+        finalResult[field] = parseFloat(finalResult[field]);
+      else finalResult[field] = parseInt(finalResult[field]);
+    }
+  });
```
Comment (Murkantilism) [inline src/views/pages/questionnaire/questions/utils.js:215]: Do we do type checking on these fields elsewhere? If not, we should add an if check on whether or not it is Numeric (something like `if(!isNaN(parseFloat(n)) && isFinite(n)) {`)

File: src/views/pages/questionnaire/Questionnaire.js [hunk 4]
```diff
@@ -1,270 +1,15 @@
...
-        </Button>
-        <Button color="primary" onClick={props.onNext} disabled={nextDisabled}>
-          {
-            ("final" in props) ? "Submit" : "Next"
-          }
-        </Button>
-      </div>
+    <div id="questionnaire-container" className="col-12 mx-auto">
+      <Route exact path="/health-report/" component={QuestionnaireMenu} />
+      <Route path="/health-report/:questionType" component={SurveyQuestions} />
```
Comment (Murkantilism) [inline src/views/pages/questionnaire/Questionnaire.js:10]: Nice refactor here especially

**Comment by Murkantilism** [comment]
@vchu22 I approved, up to you if you want to resolve my feedback within this PR or in a follow-up PR.

File: src/views/pages/questionnaire/questions/city_state_decisions.json (hunk 1 above)
Comment (vchu22) [reply src/views/pages/questionnaire/questions/city_state_decisions.json:5]: Those JSON files are form definitions generated in the embed code in Tripetto Studio. I just simply copied and pasted everything without modifying.

Comment (Murkantilism) [reply src/views/pages/questionnaire/questions/city_state_decisions.json:5]: Gotcha understood

File: src/views/pages/questionnaire/questions/utils.js (hunk 2 above)
Comment (vchu22) [reply src/views/pages/questionnaire/questions/utils.js:120]: The TODO section was a comment generated by tripetto.app. The extractAnswers is a helper function I wrote to process the raw survey response data into a cleaner Javascript object.

Comment (Murkantilism) [reply src/views/pages/questionnaire/questions/utils.js:120]: OH I see, so we're already handling the results? In that case can we delete the TODO? It's a bit misleading, I thought some work still needed to be done to handling individual pieces of the submission

Comment (vchu22) [reply src/views/pages/questionnaire/questions/utils.js:120]: I just deleted the lines

File: src/views/pages/questionnaire/questions/utils.js (hunk 3 above)
Comment (vchu22) [reply src/views/pages/questionnaire/questions/utils.js]: I just added that in a new commit

**Pull request merged**